torch/_dynamo/utils.py

```diff
@@ -2127,6 +2127,8 @@ def chromium_event_timed(
     Context manager that creates a chromium start and end event. Chromium event
     logging is integrated with dynamo_timed, so you probably want to use that
     instead. Use this context manager only if you want to avoid dynamo_timed.
+
+    Also emits profiler events so these appear in the stock profiler.
```

Comment:
I don't think we need a line break here.

We can just prefix this line with `NOTE:` like:
```
 instead. Use this context manager only if you want to avoid dynamo_timed.
 NOTE: Also emits profiler events so these appear in the stock profiler.
```